internal/utils_test.go

```diff
@@ -30,6 +30,82 @@ func assertReflectEqual(t *testing.T, a interface{}, b interface{}) {
 	}
 }
 
+func assertSelectorEqual(t *testing.T, a map[string]interface{}, b types.NamespaceSelector) {
+	t.Helper()
+
+	if !compareSelectors(a, b) {
+		t.Fatalf("%s != %s", a, b)
+	}
+}
+
+func compareSelectors(a map[string]interface{}, b types.NamespaceSelector) bool {
+	if includeA, ok := a["include"].([]interface{}); ok {
+		for i := range includeA {
+			if includeA[i].(string) != b.Include[i] {
```

Comment:
Optional: Since it's just test code, it's fine but this is assuming that `b.Include` has the same length as `includeA`.